spec/models/locker_application_spec.rb

```diff
@@ -132,11 +134,11 @@
     let!(:locker_application3) { FactoryBot.create :locker_application, complete: true }
```

Comment:
Can you add an `archived: true` case? Also, if you think it's possible the method will receive `archived` as a string, test that case as well? Hrm, it also looks like there's no `complete: false` case? Maybe throw one of those in? (sorry, I know I changed that method too)

I'm also confused by the `archived: nil` case - do you expect that to get passed to this method?

Reply:
The nil case is intended to catch if the page is loaded without query params, but it should be handled before it reaches this method. I think this test might also break if the casting in the search method is removed so this test should be removed.